pre-receive-hooks/block_self_merge_prs.sh

```diff
@@ -8,10 +8,9 @@
 # https://help.github.com/enterprise/admin/guides/developer-workflow/managing-pre-receive-hooks-on-the-github-enterprise-appliance/
 #
 
-if [ "$GITHUB_VIA" = "merge api" && "$GITHUB_PULL_REQUEST_AUTHOR_LOGIN" = "$GITHUB_USER_LOGIN"]; then
+if [[ "$GITHUB_VIA" = "merge api" ]] && [[ "$GITHUB_PULL_REQUEST_AUTHOR_LOGIN" = "$GITHUB_USER_LOGIN" ]]; then
```

Comment:
@jonico thanks for merging this while I was OOO. Looking forward to getting some other real-world pre-receive hook scripts in here 👍